<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home_window</class>
 <widget class="QDialog" name="home_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="appHeaderWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>1280</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1280</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#appHeaderWidget {
	background-color: #ffffff;
	border-bottom: 1px solid #e2e8f0;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="LB_header">
        <property name="styleSheet">
         <string notr="true">
font-family: 'Segoe UI', 'Inter', sans-serif;
font-size: 16px;
letter-spacing: 0px;
word-spacing: 0px;
color: #1e293b;
font-weight: 600;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;
padding:50px 10px;</string>
        </property>
        <property name="text">
         <string>Inventory Management</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LB_userHeader">
        <property name="styleSheet">
         <string notr="true">
font-family: 'Segoe UI', 'Inter', sans-serif;
text-align: right;
font-size: 12px;
letter-spacing: 0px;
word-spacing: 0px;
color: #64748b;
font-weight: bold;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;</string>
        </property>
        <property name="text">
         <string>Ryan Ranjbar</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_profile">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget #PB_profile {
	background-image : url(:/Images/icons/profileIcon.png);
	background-position : center;
	background-repeat : no repeat;
	background-size: contain;
	display:inline-block;
	border-radius:6px;
	cursor:pointer;
	color:#ffffff;
	font-family: 'Segoe UI', 'Inter', sans-serif;
	font-size:12px;

	padding:50px 10px;
}

QPushButton:hover {
	background-color:#f3f4f6;
}
QPushButton:pressed {
	background-color:#e2e8f0;
	color:#000000;
}

QPushButton:active {
		position:relative;
	top:1px;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="leftMenuWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>700</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>700</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget #leftMenuWidget {
	background-color: #f8fafc;
	border-right: 1px solid #e2e8f0;
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="PB_inventory">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>120</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-image : url(:/Images/icons/inventoryIcon.png);
	background-position : center;
	background-repeat : none;
	display:inline-block;
	border-radius:6px;
	cursor:pointer;
	color:#ffffff;
	font-family: 'Segoe UI', 'Inter', sans-serif;
	font-size:12px;

	padding:50px 10px;
	border: none;
    background-color: transparent;
}

QPushButton:hover {
	background-color : #f3f4f6;
	color:#000000;
}
QPushButton:pressed {
	background-color:#cbd5e1;
	color:#000000;
}

QPushButton:active {
		position:relative;
	top:1px;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="mainWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1160</width>
         <height>660</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1160</width>
         <height>660</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget #mainWidget {
	background-color : #ffffff;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
